tests/tests/swfs/avm1/define_local_with_paths/test.as

```diff
@@ -0,0 +1,80 @@
+// Hand-edited with JPEXS.
```

Comment:
Quick question: can JPEXS compile the AS back into the SWF file? Or is this purely for code review's sake?

Reply:
Yes, that's just to know what the test looks like. I would have preferred to add a .fla of course : [JPEXS generates this FLA file](https://github.com/ruffle-rs/ruffle/files/8295425/fla.zip) but I don't know if it  opens in the Flash editor and then runs and produces a similar P-code as the test I made in JPEXS.